ow_materials/src/populate_materials.cpp

```diff
@@ -0,0 +1,80 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include <sstream>
+#include <initializer_list>
+#include <vector>
+#include <set>
+#include <limits>
+
+#include <ros/ros.h>
+
+#include <populate_materials.h>
+
+using std::string, std::begin, std::end, std::initializer_list, std::set,
+      std::vector, std::stringstream, std::numeric_limits, std::runtime_error;
+
+using namespace ow_materials;
+
+static string join(const initializer_list<string> &args) {
+  stringstream joined;
+
+  for (auto x = begin(args); x != end(args) - 1; ++x)
+    joined << *x << "/";
```

Comment:
Suggest bracketing for safety.  Same for one-line IF statements in rest of file.

Reply:
done